tests/libpod/exec.go

```diff
@@ -0,0 +1,90 @@
+/*
+ * This file is part of the KubeVirt project
```

Comment:
@qinqon FYI, I have a similar [PR](https://github.com/kubevirt/kubevirt/pull/7944/files#diff-71907393fa166555d5d189a3d1cd83b636c5d5d6e4ce119afa688c14d8d5748b) opened almost 2 months ago to move execute functions from utils. It was reviewed and suggested by @itamarh to have those functions refactored into `ExecuteCommandOnPod` and `ExecuteCommandOnPodWithOptions` (I have some bug there as it causes some tests to fail for some unknown reason).

Reply:
@codingben let's continue at your PR I am going to close this one since it's too big.